enclave-tls/README.md

```diff
@@ -55,6 +55,10 @@ Right now, Enclave TLS supports the following instance types:
 
 By default,  Enclave TLS will select the **highest priority** instance to use.
 
+The following instances matrix will introduce possible combinations between each instances.
```

Comment:
Change crypt to "Crypt Wrapper".
Change Encalve Quotes to "Attester" and "Verifier".

Reply:
ok